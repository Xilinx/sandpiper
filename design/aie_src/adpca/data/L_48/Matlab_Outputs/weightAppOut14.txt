1.075585938e+01 2.688363838e+01
1.441906929e+00 5.902635574e+00
-3.849196196e+00 -1.431148052e+00
