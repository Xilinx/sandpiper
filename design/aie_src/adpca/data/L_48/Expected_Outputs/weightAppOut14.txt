1.075483894e+01 2.688263512e+01 
1.441799641e+00 5.902520180e+00 
-3.849082708e+00 -1.431080103e+00 
